addons/mail/models/mail_thread.py

```diff
@@ -3788,6 +3788,18 @@ def _notify_by_web_push_prepare_payload(self, message, msg_vals=False):
         else:
             icon = '/web/static/img/odoo-icon-192x192.png'
 
+        if not body and not message.body and message.attachment_ids:
+            total_attachments = len(message.attachment_ids)
+            first_attachment = message.attachment_ids[0]
+            first_attachment_label = _('Voice message') if first_attachment.voice_ids else first_attachment.name
+
+            body = (
+                first_attachment_label
+                if total_attachments == 1
+                else _("%(attachment_name)s and %(count)s other attachment(s).",
```

Comment:
This should be in the language of the target user, not the language of the sending user (more generic issues with all push notifications, that should contain user-specific content).